spec/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
- [ ] All the custom methods in the user model should have unit tests, kindly make sure to test the `last_three_posts` method and all other methouds in the model